### ISSUE TITLE
security: os.system() f-string interpolation in scripts/finetune-b00t.py — command injection via B00T_OUTPUT_DIR

### ISSUE THREAD

**Issue by elasticdotventures** [elasticdotventures/dotfiles#531]
## Finding

`scripts/finetune-b00t.py` (PR #528) contains at least two `os.system(...)` calls with f-string paths derived from the `B00T_OUTPUT_DIR` environment variable:

\`\`\`python
os.system(f"uv pip install {' '.join(packages)}")
# and
os.system(f"... {merged_path} ...")
\`\`\`

If `B00T_OUTPUT_DIR` contains spaces or shell metacharacters (e.g. \`; rm -rf ~\`), the command executes attacker-controlled shell code.

## Fix

Replace all `os.system(f"...")` calls with `subprocess.run([...])` using list arguments:

\`\`\`python
# Instead of:
os.system(f"some-cmd {path}")
# Use:
subprocess.run(["some-cmd", str(path)], check=True)
\`\`\`

## Broader context

The existing `fine-tune/train_unsloth.py` avoids this pattern entirely. `scripts/finetune-b00t.py` is a parallel reimplementation of lower quality — see DRY issue #532.

## Labels

security, pr-528, finetune

**Comment by elasticdotventures** [comment]
Resolved by #532 — scripts/finetune-b00t.py was removed (duplicated finetune scripts subsumed by fine-tune/ pipeline in commit 884c90f). The replacement fine-tune/train_unsloth.py avoids os.system() entirely.
